Tasks/JenkinsQueueJobV2/jobsearch.ts

```diff
@@ -43,23 +43,23 @@ export class JobSearch {
         if (!thisSearch.Initialized) { //only initialize once
             const apiTaskUrl: string = util.addUrlSegment(thisSearch.taskUrl, '/api/json?tree=downstreamProjects[name,url,color],lastBuild[number]');
             tl.debug('getting job task URL:' + apiTaskUrl);
-            request.get({ url: apiTaskUrl, strictSSL: thisSearch.queue.TaskOptions.strictSSL }, function requestCallBack(err, httpResponse, body) {
+
+            var request = new WebRequest();
```

Comment:
Prefer `const` here and below

Reply:
I agree w/ Brian.  I can't think of a reason why we should ever introduce a `var` into our codebase.  Use `const` or `let` if necessary, if we need to use `var`, we should refactor instead.